### ISSUE TITLE
Sign in problems, help problems, video upload problems

### ISSUE THREAD

**Issue by oneeve8** [steemit/steem#2196]
Here is my original comment that was closed:
“And by the way, there is no contact information within your site for outside users. I found this github by searching online and also found a comment from someone else with your contact info:
Contact@steemit.com”

The supposed “help” that I got was to go to 
Steemit.chat 
I signed up for the chat
It’s full of people having the same problem that I am signing up
There is nobody there to help, it’s all people either waiting to sign up, having problems signing up, lost their username or password or there phone # is locked up because it’s “waiting for verification”

My other issue was also closed- told the same thing... to go to Steemit chat
My problem is that...
I got my code on my phone but there is nowhere to put the code to verify
If I try to log in it says the user name is not set up yet
If I try to start over again it says the phone number is in the process of being verified
Yes, Of course it is, because it is MY phone number... I got the code so where do I put it?

In the chat there is someone also saying they can’t upload videos to DTube anyway, which is why I wanted to join. I’m having the same problem they mentioned: the file format is not supported. Which is simply an MP4 that uploads fine to youtube. I have seen that other users have also been having trouble loading videos and have moved to Bitchute. But Bitchute has the same issue. So now what?

Now you are going to close this “issue” and tell me to go to Steemit chat again?

How about explaining what the problem is so at least people know what is going on. There is nobody in the chat that has answers, only questions. But if you don’t get the video uploading fixed I guess don’t bother helping me sign up anyway. But at least you could say something about what is going on so people are not left hanging.
Thank you.

**Comment by TimCliff** [comment]
At the top of the help channel as soon as you join, it tells you to read the pinned messages. There is info there. It also says that the channel is supported by volunteers, and it may take 24 hours to receive a response.

**Comment by oneeve8** [comment]
Yeah, it doesn’t look like anybody is getting help or answers there. It might be helpful for somebody at Steemit to point people to known issues or have a list of known issues so that people are not left wondering. But thanks anyway. I looked on Steemit faucet and my issue was listed by someone else having the same issue and I posted my comment there as well and soon after somebody marked it as a bug. I hope it gets fixed. It sounds very promising, and a lot of people want to support what you are doing including myself, but I guess a lot of bugs still. I think informing people is better than leaving people hanging. People do want alternatives to other social media and they like the block chain concept. Thanks for your time.
